app/src/androidTest/java/ch/epfl/culturequest/notifications/FollowNotificationTest.java

```diff
@@ -34,11 +37,13 @@ public void setup() {
     @Test
     public void FollowNotificationIsCorrectlyCreated() {
         Notification followNotification = new FollowNotification("Follower").buildNotification(context);
-        assertThat(followNotification.extras.get(Notification.EXTRA_TITLE).toString(), is(profile.getUsername() + ", you have a new follower!"));
-        assertThat(followNotification.extras.get(Notification.EXTRA_TEXT).toString(), is("Follower is now following you!"));
+        assertThat(followNotification.extras.get(Notification.EXTRA_TITLE).toString(), is("Follower, you have a new follower!"));
+        assertThat(followNotification.extras.get(Notification.EXTRA_TEXT).toString(), is(profile.getUsername() + " is now following you!"));
         assertThat(followNotification.priority, is(Notification.PRIORITY_DEFAULT));
         assertThat(followNotification.getSmallIcon().getResId(), is(R.drawable.logo_compact));
         assertThat(followNotification.getChannelId(), is(FollowNotification.CHANNEL_ID));
+        Intent intent = new Intent(context, DisplayUserProfileActivity.class).setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK).putExtra("username", profile.getUid());
+        assertThat(followNotification.contentIntent, is(PendingIntent.getActivity(context, 0, intent, PendingIntent.FLAG_IMMUTABLE)));
```

Comment:
Shouldn't the name of the extra here should be "uid" instead "username" ?

Reply:
Yes indeed, thanks!